testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -1296,11 +1296,5 @@ def test_empty_creation_raises_error(self):
 
 def test_deprecate_b_tempfactors():
     u = mda.Universe(PDB)
-    with pytest.warns(DeprecationWarning, match="alias"):
+    with pytest.warns(DeprecationWarning, match="use the tempfactor"):
         u.add_TopologyAttr("bfactors")
```

Comment:
Is it worth testing here that the setter works properly? I.e. that tempfactors are set?